join.md

```diff
@@ -6,42 +6,39 @@ permalink: /subscribe/
 
 # Join us
 
-Our Slack is governed by the principles and rules in our [Community Guide](/community-guide). By joining, you agree to follow them.
+1. Fill out this form to subscribe to our newsletter.
+2. If eligible, you will be invited to our Slack.
+3. You agree to the [Community Guide](/community-guide) by filling the form.
 
-<h3 class="marg-b-3">Please provide the following:</h3>
+<h3>Please provide the following:</h3>
 <form class="join-form" method="POST" target="_blank" class="marg-b-4" data-netlify="true" action="/welcome" netlify-honeypot="bot-field">
   <label style="display:none">
     Don’t fill this out if you’re human: <input name="bot-field" />
   </label>
-  <label class="marg-b-3" for="email">
-    <div><b>Email:</b></div>
+  <label for="name">
+    <div><b>Your full name</b></div>
+    <input id="name" type="text" required name="name">
+  </label>
+  <label for="email">
+    <div><b>Email</b></div>
     <input id="email" type="email" required name="email">
   </label>
-  <label class="marg-b-3" for="name">
-    <div><b>Name:</b></div>
-    <input id="name" type="text" required name="name">
+  <label for="social">
+    <div><b>Social media link #1</b></div>
```

Comment:
```suggestion
    <div><b>Social media link #1 (e.g. LinkedIn, Bluesky, Twitter)</b></div>
```
accessibility nitpick: placeholders should not contain explanations of the form input
(https://www.deque.com/blog/accessible-forms-the-problem-with-placeholders/)

Reply:
Having read the a11y documentation, I don't think it applies here. If the placeholders were completely removed the labels would still provide adequate instructions. Optional fields are listed as option within label, and not within placeholder for example. When users submit they will get more specific validation (missing fields, nothing beyond that)